### DISCUSSION TITLE
Styling of nested tables - is inheritance bug or feature?

### DISCUSSION THREAD

**Discussion by mrljsh** [icflorescu/mantine-datatable#488] [General]
I've noticed a change in behavior with Mantine DataTable V7 where nested tables now inherit styles from the parent component. This is different from the behavior in V6 where nested tables did not inherit styles. I'm unsure if this is intentional or a bug.

Is there a way to configure or override this behavior? Specifically, I'd like to explore the possibility of making children not use inherited styles but instead default to styles defined in mantine-datatable/styles.

**Comment by dayreon** [comment]
Same question
